vncdotool/command.py

```diff
@@ -393,6 +393,9 @@ def vncdo():
         metavar='FACTOR', default=1.0,
         help='pause time is accelerated by FACTOR [x%default]')
 
+    op.add_option('--incremental-expect', action='store', type='int', metavar='INCREMENTAL_EXPECT', default='1',
+        help='try setting this to 0 if expect has problems matching the current screen [%default]')
```

Comment:
Originally, I made incremental=1 the default because it seemed more compatible, https://github.com/sibson/vncdotool/commit/28c15b65305ecc010a2cd0b708667cb489e83568. This feels like its a leaking abstraction and exposing protocol level features in the UX.  Is there a better name we can use here to help users understand what the feature is and why they might use it?  ```--disable-incremental-refreshes``` is the first thing that comes to mind but I think it suffers the same problem and perhaps implies a wider scope of impact.

Reply:
> This feels like its a leaking abstraction and exposing protocol level features in the UX. Is there a better name we can use here to help users understand what the feature is and why they might use it?

I guess you are right but this is *vnc*dotool and exposing vnc features should be fine for some amount?

> `--disable-incremental-refreshes` is the first thing that comes to mind but I think it suffers the same problem and perhaps implies a wider scope of impact.

I don't know, maybe `--expect-full-frames` or `--request-full-frames`? It seems a little bit hard to understand though?